src/main/scala/devsearch.lookup/RawDB.scala

```diff
@@ -0,0 +1,22 @@
+package devsearch.lookup
+
+import reactivemongo.api.MongoDriver
+import reactivemongo.api.collections.default.BSONCollection
+import reactivemongo.bson.BSONDocument
+import reactivemongo.core.commands.RawCommand
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+object RawDB {
+  val DB_SERVER = "localhost"
+  val DB_NAME = "devsearch"
+
+  val driver = new MongoDriver
```

Comment:
I didn't know that Scala objects are lazy. I guess that works then. Except that singletons are bad. :P